backend/app/views/spree/admin/prices/_table.html.erb

```diff
@@ -0,0 +1,53 @@
+<%= paginate prices, theme: "solidus_admin" %>
+
+<table class="index prices">
+  <colgroup>
+    <col style="width: 15%" />
+    <col style="width: 35%" />
+    <col style="width: 35%" />
+    <col style="width: 15%" />
+  </colgroup>
+
+  <thead data-hook="prices_header">
+    <tr>
+      <th><%= Spree::Variant.model_name.human %> </th>
+      <th><%= Spree::Price.human_attribute_name(:is_default) %></th>
+      <th><%= Spree::Price.human_attribute_name(:amount) %></th>
+      <th><%= Spree::Price.human_attribute_name(:currency) %></th>
+      <th class="actions"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+  <% prices.each do |price| %>
+    <tr data-hook="prices_row" class="<%= "deleted" if price.deleted? %> <%= cycle('odd', 'even')%>">
```

Comment:
Would be great if we could follow rails convention here and put this into a `_price` partial and render the `@prices` collection.